db/Pos/DB/Functions.hs

```diff
@@ -101,7 +113,7 @@ dbDecodeMaybeWP s
 encodeWithKeyPrefix
     :: forall i . (DBIteratorClass i, Bi (IterKey i))
     => IterKey i -> ByteString
-encodeWithKeyPrefix = (iterKeyPrefix @i <>) . serialize'
+encodeWithKeyPrefix = (iterKeyPrefix @i <>) . dbSerialize
```

Comment:
@gromakovsky Please take a look at this line when you review (encoding keys)

Reply:
@rockbmb It's part of serialized key, so it shouldn't have a version as was clarified by @georgeee.
Also I propose to rename `dbSerialize` to `dbSerializeValue` to make it clear that such serialization should be used only for values.